Assets/HoloToolkit/ARCA/Images/share-icon-apple.png.meta

```diff
@@ -0,0 +1,74 @@
+fileFormatVersion: 2
```

Comment:
What was the source of the .png file associated with this .meta file (I cannot comment on the png in this diff)?

Reply:
@davidkline-ms , is your question, "Is [this](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/1884/files/19d353a2614981f4ea38f0846df0957648bfe2f1#diff-b9a3e3ecf7d3a593d26e238633bb56d8) Apple's icon and is OK for MRTK to distribute it under the MIT license?"

Reply:
We will change the name of the file. The “Apple” part is unrelated to Apple

Reply:
@timGerken, Yes. I was asking if this was a 3rd party file (eg Apple) and whether it was allowed to be redistributed.